src/org/labkey/test/components/react/MultiMenu.java

```diff
@@ -40,48 +40,73 @@ private WebElement getMenuList()
         return Locator.tagWithClass("ul", "dropdown-menu").findElement(this);
     }
 
+    /*
+        finds the first menu item with the specified text
+     */
     public WebElement getMenuItem(String text)
     {
-        return Locator.tag("li").withChild(Locator.tagWithAttribute("a", "role", "menuitem")
-                    .withText(text)).waitForElement(getMenuList(), _menuWaitTmeout);
+        return Locators.menuItem().withText(text).waitForElement(getMenuList(), _menuWaitTmeout);
     }
 
+    /**
+     * Find the first item with specified text under the path of the
+     * @param menuText Text of the menu to expand
+     * @param itemText  Text of the item to find under its path
+     * @return
+     */
+    public WebElement getMenuItem(String menuText, String itemText)
+    {
+        return getItemUnderToggle(menuText, itemText);
+    }
+
+    /**
+     * Checks whether the specified menu item is disabled
+     * @param text Text of the menu item
+     * @return  true if class contains 'disabled'
+     */
     public boolean isMenuItemDisabled(String text)
     {
         return getMenuItem(text).getAttribute("class").toLowerCase().contains("disabled");
     }
 
+    public boolean isMenuItemDisabled(String menuText, String itemText)
+    {
+        return getItemUnderToggle(menuText, itemText).getAttribute("class").toLowerCase().contains("disabled");
```

Comment:
Does `WebElement.isEnabled` work for these?
```suggestion
        return !getItemUnderToggle(menuText, itemText).isEnabled();
```

Reply:
Let's find out!

Reply:
Sadly, I found out that webElement.isEnabled can't tell the difference between having 'disabled' in the class and not.
restoring use of class to determine disabled for now
I'll file a backlog issue to track our problem of having lots of non-standard enable/disable implementations in our products